frontend/test/tickets/tickets.sagas.spec.ts

```diff
@@ -53,6 +53,47 @@ describe('Tickets: sagas', () => {
 					.silentRun();
 			})
 
+			it('should call fetch containers Ticket endpoint', async () => {
+				const ticket = tickets[0];
+				mockServer
+					.get(`/teamspaces/${teamspace}/projects/${projectId}/containers/${modelId}/tickets/${ticket._id}`)
+					.reply(200, ticket);
+
+				await expectSaga(TicketsSaga.default)
+					.dispatch(TicketsActions.fetchTicket(teamspace, projectId, modelId, ticket._id, false))
+					.put(TicketsActions.upsertTicketSuccess(modelId, ticket))
+					.silentRun();
+			})
+			
+			it('should call container`s update ticket endpoint', async () => {
+				const ticket = tickets[0];
+				mockServer
+					.patch(`/teamspaces/${teamspace}/projects/${projectId}/containers/${modelId}/tickets/${ticket._id}`)
+					.reply(200);
+
+				const updateProp = {title:'updatedContainerTicketName'};
+
+				await expectSaga(TicketsSaga.default)
+					.dispatch(TicketsActions.updateTicket(teamspace, projectId, modelId, ticket._id, updateProp, false))
+					.put(TicketsActions.upsertTicketSuccess(modelId, updateProp))
+					.silentRun();
+			})
+
+			it('should call container`s create ticket endpoint', async () => {
+				const newticket = ticketMockFactory();
+				delete newticket._id;
```

Comment:
you can write this as

```ts
const { _id, ...newTicket } = ticketMockFactory();
```

(Just a suggestion, not a change request)

Reply:
It complicates the use of _id in the rest of the function because it gets defined as a const.